doc/rho.1

```diff
@@ -0,0 +1,408 @@
+.TH rho 1 "August 10, 2017" "version 0.1" "rho User Commands"
+.SH NAME
+rho \- Easily discover and manage systems on your network.
+
+.SH SYNOPSIS
+rho
+.I command subcommand [options]
+
+.SH DESCRIPTION
+rho is a network discovery tool to identify the number of systems running on a network, their operating system, hardware, and release numbers for almost any Linux or Unix flavor. Being able to identify the systems running on the network is a vital component to managing licenses and renewals. Ultimately, discovery is part of the larger sysadmin task of managing inventories.
+.PP
+rho uses three configuration entries to manage the discovery process.
+.I Profiles
+define the network or subnet that is being monitored.
+.I Credentials
+contains the usernames, passwords or, alternatively, the SSH keys of the user as which discovery runs. There can be multiple network profiles and authentication credentials, used in any combination.
```

Comment:
I suggest we change "as which discovery runs" to "which discovery runs as". The common objection to ending sentences with preposition is based on a myth: http://blog.oxforddictionaries.com/2011/11/grammar-myths-prepositions/